testing/testcafe/tests/dataGrid/headerFilter.ts

```diff
@@ -27,3 +29,26 @@ test('The header filter should fit inside the viewport if the grid is scrolled h
   height: 400,
   headerFilter: { visible: true },
 }));
+
+test('HeaderFilter popup screenshot', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const dataGrid = new DataGrid('#container');
+
+  await t
+    .click(dataGrid.getHeaders().getHeaderRow(0).getHeaderCell(0).getFilterIcon())
```

Comment:
Let's move the element that we want to click to a separate variable:

```suggestion
  const dataGrid = new DataGrid('#container');
  const filterIconElement = dataGrid.getHeaders().getHeaderRow(0).getHeaderCell(0).getFilterIcon();

  await t
    .click(filterIconElement)
```